.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  'extends': 'airbnb',
+  'parser': 'babel-eslint',
+  'env': {
+    'jest': true,
+  },
+  'rules': {
+    'no-use-before-define': 'off',
+    'react/jsx-filename-extension': 'off',
+    'react/prop-types': 'off',
+    'comma-dangle': 'off'
+  },
+  'globals': {
+    "fetch": false
```

Comment:
React native has fetch. Lets change this to true.

Reply:
Okay, updating now.